spacy/training/iob_utils.py

```diff
@@ -218,6 +218,10 @@ def tags_to_entities(tags: Iterable[str]) -> List[Tuple[str, int, int]]:
     return entities
 
 
+def strip_bilu_prefix(label: str):
```

Comment:
We might want to add the type annotation for the result too, just for the sake of completeness.